Extensions/alternate_audio_player.js

```diff
@@ -0,0 +1,60 @@
+//* TITLE Alternate Audio Player **//
```

Comment:
I'll change it to Vanilla Audio. It doesn't do quite the same thing (yet?) though since it adds a new player, rather than totally replacing Tumblr's.